src/routes/__error.svelte

```diff
@@ -8,34 +8,52 @@
 </svelte:head>
 
 <section>
-  <img src="/assets/png/404-h.png" class="wide" alt="404 - Page Not Found" />
-  <img src="/assets/png/404-v.png" class="mobile" alt="404 - Page Not Found" />
+  <em>404</em>
+  <h1>Frank can't find where you're going!</h1>
+  <img src="/assets/png/lost-frank.png" class="wide" alt="404 - Page Not Found" />
+  <img src="/assets/png/lost-frank.png" class="mobile" alt="404 - Page Not Found" />
```

Comment:
I noticed that the `wide` and `mobile` class names are not in us, so you can go ahead and remove one of the images.
```suggestion
  <img src="/assets/png/lost-frank.png" alt="404 - Page Not Found" />
```

Reply:
done